src/sidebar/cross-origin-rpc.js

```diff
@@ -38,28 +53,25 @@ function start(store, settings, $window) {
 
   /** Return a JSON-RPC response to the given JSON-RPC request object. */
   function jsonRpcResponse(request) {
-    // The set of methods that clients can call.
-    let methods = {
-      searchUris: store.searchUris,
-    };
-
-    let method = methods[request.method];
-
-    let response = {
-      jsonrpc: '2.0',
-      id: request.id,
-    };
+    // Find the mapped method requested
+    const method = methods[request.method];
 
-    if (method) {
-      response.result = method();
-    } else {
-      response.error = {
-        code: -32601,
-        message: 'Method not found',
+    // Return an error response if the method name is invalid.
+    if (method === undefined) {
```

Comment:
copied from lms

Reply:
Is it more idiomatic to use `typeof` here at least?

Reply:
I vote for this since its less chars and its checking type, but I see that typeof is used a lot in the client. If you're not picky, I prefer this since its less characters and does not need to perform a string check.

Reply:
I vote for this since its less chars and its checking type, but I see that typeof is used a lot in the client. If you're not picky, I prefer this since its less characters and does not need to perform a string check.